playwright.config.ts

```diff
@@ -8,22 +8,22 @@ import { TestOptions } from "./tests/fixtures/lxd-test";
 const config: PlaywrightTestConfig<TestOptions> = {
   testDir: "./tests",
   /* Maximum time one test can run for. */
-  timeout: 120 * 1000,
+  timeout: 120_000,
   expect: {
     /**
      * Maximum time expect() should wait for the condition to be met.
      * For example in `await expect(locator).toHaveText();`
      */
-    timeout: 120 * 1000,
+    timeout: 120_000,
   },
   /* Run tests in files in parallel */
   fullyParallel: true,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
-  /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : 3,
+  /* Opt out of parallel tests. */
+  workers: 1,
```

Comment:
we don't want to have a bit of a speed boost for local testing? or this parallel workers break the coverage test?